src/globus_sdk/services/flows/client.py

```diff
@@ -487,6 +487,47 @@ def delete_flow(
 
         return self.delete(f"/flows/{flow_id}", query_params=query_params)
 
+    @paging.has_paginator(paging.MarkerPaginator, items_key="runs")
+    def list_runs(
+        self,
+        *,
+        marker: str | None = None,
+        query_params: dict[str, t.Any] | None = None,
+    ) -> IterableRunsResponse:
```

Comment:
There's a lot of filter query params (`filter_status`, `filter_roles`, `filter_role`, `filter_label` ,`filter_flow_id`, `filter_completion_time`, `filter_start_time`, `filter_flow_title`) allowed for this route.

I'm concerned about omitting the existence of these filters from all forms of sdk documentation. A lot of them are useful & users will only discover them if they navigate through the sdk docs, click on the api info tab, and follow the link to a separate website which documents the api info.

That's a really high level of effort required for a user to discover that they could filter runs by, for instance, the time the runs were started. Especially when we have customers (APS) who do have 1000s of runs that I could very much envision them wanting to filter through.

Reply:
Just to put a bow on this: per our conversation, we're going to pull in only `filter_flow_id`.
It's the only one of these which we feel _somewhat_ confident _as of today_ offers value to keep even in a future in which we push users to query in Search more actively.

Reply:
I've updated with the addition of `filter_flow_id`, and took the opportunity to try a slightly different approach to testing the parameter passing behavior from what I've done in the past.

Rather than checking the request parameters directly, the test actually asserts that the filters have the desired effect on an orchestrated `ResponseList` with a few appropriately filter-able responses.